crypto/evp/digest.c

```diff
@@ -70,6 +70,42 @@ int EVP_MD_CTX_reset(EVP_MD_CTX *ctx)
     return 1;
 }
 
+EVP_MD_CTX *evp_md_ctx_new_ex(EVP_PKEY *pkey, const ASN1_OCTET_STRING *id,
+                              OPENSSL_CTX *libctx, const char *propq)
+{
+    EVP_MD_CTX *ctx;
+    EVP_PKEY_CTX *pctx = NULL;
+
+    if ((ctx = EVP_MD_CTX_new()) == NULL
+        || (pctx = EVP_PKEY_CTX_new_from_pkey(libctx, pkey, propq)) == NULL) {
+        X509err(0, ERR_R_MALLOC_FAILURE);
+        goto err;
+    }
+
+#ifndef OPENSSL_NO_SM2
```

Comment:
I'm not sure why this is required?

Reply:
I inferred this from a failed Travis run, but likely this is not needed anymore, see below...